src/index.test.ts

```diff
@@ -3,11 +3,11 @@ import * as path from 'path';
 
 const DEFAULT_CONFIG = path.join(__dirname, 'fixtures', 'tslint.json');
 const CONFIG_WITH_JS = path.join(__dirname, 'fixtures', 'tslint-with-js.json');
-
+const CONFIG_WITH_ERROR = path.join(__dirname, 'fixtures', 'tslint-with-errors.json');
 
```

Comment:
<!--
  7 failure:  " should be ' lin..., " should be ' lin..., " should be ' lin..., " should be ' lin..., Unnecessarily quo..., Unnecessarily quo..., tab indentation e...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: src/index.test.ts;
  Line: 7;
-->
- :no_entry_sign: " should be ' line - tasks.json (quotemark)
- :no_entry_sign: " should be ' line - tasks.json (quotemark)
- :no_entry_sign: " should be ' line - tslint-with-errors.json (quotemark)
- :no_entry_sign: " should be ' line - tslint-with-errors.json (quotemark)
- :no_entry_sign: Unnecessarily quoted property 'quotemark' found. line - tslint-with-errors.json (object-literal-key-quotes)
- :no_entry_sign: Unnecessarily quoted property 'script' found. line - tasks.json (object-literal-key-quotes)
- :no_entry_sign: tab indentation expected line - tasks.json (indent)